### PR TITLE
Translate to spanish: 023 Bash Redirection

### PR DESCRIPTION
**Pull request by marcelozarate**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

This translates chapter 023 to spanish in a formal tone like the rest of the book.

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

File: ebook/es/content/023-bash-redirection.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+## Resumen
+|**Operador** |**Descripción** |
+|:---|:---|
+|`>`|`Guardar la salida a un archivo`|
+|`>>`|`Añadir la salida a un archivo`|
+|`<`|`Leer una entrada desde un archivo`|
+|`2>`|`Redirigir mensajes de error`|
+|`\|`|`Enviar la salida de un programa como entrada a otro`|
+|`<<`|`Canalizar múltiple líneas a un programa limpiamente`|
+|`<<<`|`Canalizar una única línea a un programa limpiamente`|
```
Comment (marcelozarate) [inline ebook/es/content/023-bash-redirection.md:223]:
```suggestion
|`<<<`|`Canalizar una única línea a un programa limpiamente`|

```

**Review by marcelozarate — COMMENTED** [review]
Added line break at the end

**Comment by marcelozarate** [comment]
Can you check this out? @bobbyiliev

**Review by bobbyiliev — APPROVED** [review]
Looks good to me! Thank you!

**Pull request merged**